conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+    * UXD user research
+    * Pain points shared from internal feedback sessions (Solution Architects, Technical Marketing, CEE/Customer Support, etc.)
+    * Product Management input
+    * OpenShift or Kubernetes documentation
+    * Features we want to highlight or push more to entice users to try
+    * Areas we want to offer more opinionated views or flows. For example, there are likely cases where there are multiple ways for users to accomplish a task, but we’re recommending a method or suggesting how to get up and running with XYZ.
+  * Note that regardless of where the quick start idea originates, quick starts should always be a part of an epic.
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
+  * **Time frame:** Before epic prioritization
+  * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start. Epics are created in the relevant development project.
+2. Plan user flows and interaction design. (Optional)
+  * UXD Designer, UXD Content Strategist, and CCS Content Strategist and/or Writer meet with Product Manager to determine the quick start’s goal and where we need to reduce friction or confusion in completing some tasks.
+  * Collaborate on what the ideal user flow would look like. For example, what are the tasks and the sub tasks that users need to do in order to complete the quick start?
+  * Work with Abi to figure out who we work with in CCS and tag in JIRA story.
+    * Consider prioritization (track in JIRA); use template.
+    * Maybe need a separate process for external teams; have UXD Content & CCS just do a quick run-through.
+  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs.
+  * **Team responsible:** UXD Design (including Dev Tools UXD); Product Management; UXD Content Strategy; CCS
+  * **Time frame:** Conceptual design story (Sprint 1)
+  * **Definition of done:**
+    * UXD Designer creates interaction designs (not hi-fi) in Sketch and Marvel.
+    * Product Manager reviews the interactions and provides feedback.
+    * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 2: First draft
+The contributing team writes the first draft of the quick start, using the following as references:
+* Interaction design plans (to align with user flow)
+* Technical documentation (to avoid contradiction and duplication of effort)
+* Red Hat brand standards/PatternFly UX writing style guide (to align with brand voice and UX writing best practices)
+* CCS style resources (to align with docs best practices)
+* Quick start writing template (to maintain consistency)
+* **Team responsible:** Contributing team; UXD team
```

Comment:
These bullets are blending with the rest of the content. Can we add a break before "Team responsible" ?